src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -606,7 +622,23 @@ void parse_cache_info_linux(const std::vector<std::vector<std::string>> system_i
     }
 
     for (size_t n = 0; n < offline_list.size(); n++) {
-        _cpu_mapping_table.erase(_cpu_mapping_table.begin() + offline_list[n] - n);
+        const int erase_index = offline_list[n] - static_cast<int>(n);
+        if (erase_index < 0 || static_cast<size_t>(erase_index) >= _cpu_mapping_table.size()) {
```

Comment:
Please share the system_info_table which will get error on master branch and can be handled by this change.